### DISCUSSION TITLE
Added MQTT to HA but not sure where to go now

### DISCUSSION THREAD

**Discussion by chesterflaps** [tsightler/ring-mqtt#583] [Support Q&A]
Evening all.

I am VERY new to HA and my lord I have started on one hell of an adventure.

I have the Ring alarm system and use all the motion sensors and doors sensors to control my lights. I have followed the steps to get the ring sutomation setup using the MQTT github repository but I cannot see where I go from here. I signed in to my ring account via MQTT and "Ring-MQTT with Video Streaming" is showing in the list of devices but none of my sensors are or my alarm system in general.

Can anyone offer me any advice please?

**Comment by chesterflaps** [comment]
Logs saying this

2023-03-12T20:45:36.439Z ring-mqtt Attempting to reconnect to MQTT broker...
2023-03-12T20:45:36.446Z ring-mqtt Unable to connect to MQTT broker connect ECONNREFUSED 172.30.32.1:1883

**Comment by jeffgraham-hub** [comment]
I'm no expert but the first place I would start is looking at the logs. It should give you an idea of what is failing.JeffOn Mar 12, 2023 3:41 PM, chesterflaps ***@***.***> wrote:
Evening all.
I am VERY new to HA and my lord I have started on one hell of an adventure.
I have the Ring alarm system and use all the motion sensors and doors sensors to control my lights. I have followed the steps to get the ring sutomation setup using the MQTT github repository but I cannot see where I go from here. I signed in to my ring account via MQTT and "Ring-MQTT with Video Streaming" is showing in the list of devices but none of my sensors are or my alarm system in general.
Can anyone offer me any advice please?

**Comment by jeffgraham-hub** [comment]
You do have a MQTT broker installed correct?

**Reply by chesterflaps** [reply]
I installed Ring-MQTT with Video Streaming

**Reply by jeffgraham-hub** [reply]
You need to also install a MQTt broker like Mosquito broker. That is what ring-mqtt is trying to connect to. You can add it under addons in HA.

**Reply by chesterflaps** [reply]
Thank you so much!! Did that, refreshed the ring mqtt and BOOM all working. I love the HA community already and I'm sure I will be back very soon with more questions, ha ha.
